docker/.env

```
@@ -239,6 +246,12 @@ SMQ_USERS_HTTP_HOST=users
 SMQ_USERS_HTTP_PORT=9002
 SMQ_USERS_HTTP_SERVER_CERT=
 SMQ_USERS_HTTP_SERVER_KEY=
+SMQ_USERS_GRPC_HOST=users
+SMQ_USERS_GRPC_PORT=7002
+SMQ_USERS_GRPC_SERVER_CERT=
+SMQ_USERS_GRPC_SERVER_KEY=
+SMQ_USERS_GRPC_SERVER_CA_CERTS=
+SMQ_USERS_GRPC_CLIENT_CA_CERTS=
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The SMQ_USERS_GRPC_CLIENT_CA_CERTS key is duplicated

(DuplicatedKey)